homeassistant/components/powerwall/sensor.py

```diff
@@ -124,9 +142,47 @@ def extra_state_attributes(self):
         meter = self.coordinator.data[POWERWALL_API_METERS].get_meter(self._meter)
         return {
             ATTR_FREQUENCY: round(meter.frequency, 1),
-            ATTR_ENERGY_EXPORTED: meter.get_energy_exported(),
-            ATTR_ENERGY_IMPORTED: meter.get_energy_imported(),
             ATTR_INSTANT_AVERAGE_VOLTAGE: round(meter.average_voltage, 1),
             ATTR_INSTANT_TOTAL_CURRENT: meter.get_instant_total_current(),
```

Comment:
Side note: We have sensor device classes for current and voltage so those attributes may want to move to separate sensors too.

Also frequency and is active should probably be separate sensor and binary sensor entities.

Reply:
I agree. I thought about doing that in this PR, but I wanted to keep it smaller since I've got a lot of spinning plates at the moment.